aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -871,13 +914,33 @@ func resourceAviatrixTransitGatewayCreate(d *schema.ResourceData, meta interface
 		gateway.Interfaces = b64.StdEncoding.EncodeToString(interfaceList)
 
 		if goaviatrix.IsCloudType(cloudType, goaviatrix.EDGENEO) {
-			// set the static interface mapping for AEP
-			interfaceMapping := map[string][]string{
-				"eth0": {"wan", "0"},
-				"eth1": {"wan", "1"},
-				"eth2": {"wan", "2"},
-				"eth3": {"mgmt", "0"},
-				"eth4": {"wan", "3"},
+			/*
+				TODO: Add support for user provided interface mapping. Currently, the interface mapping is hardcoded for Dell devices. This will be updated in a future release.
+			*/
+			interfaceMapping := map[string][]string{}
+			interfaceMappingInput := d.Get("interface_mapping").([]interface{})
+			if interfaceMappingInput != nil {
+				// Set the interface mapping for ESXI devices
```

Comment:
Comments are conflicting, ESXi vs Dell

Reply:
In this implementation, if the user provides the interface mapping then we are assuming its for internal testing on ESXI devices but for production use case, no input is expected from the user and the interface mapping is set for Dell devices by default.

Reply:
Updated the comment